db/seeds/user_seeds.rb

```diff
@@ -0,0 +1,78 @@
+def users_details
+  [
+    {
+       username: "KingKevin", 
+       email: "kevin@rails.com"
+    },
+    {
+       username: "RockinRob", 
+       email: "rob@rails.com"
+    },
+    {
+      username: "CunningCody", 
+      email: "cody@rails.com"
+    },
+    {
+      username: "RecalcitrantRyan", 
+      email: "ryan@rails.com"
+    },
+    {
+      username: "LuckyLeo", 
+      email: "leo@rails.com"
+    },
+    {
+      username: "AbhorrentAustin", 
+      email: "austin@rails.com"
+    },
+    {
+      username: "AmazingArun", 
+      email: "arun@rails.com"
+    },
+    {
+      username: "CoolChad", 
+      email: "chad@rails.com"
+    },
+    {
+      username: "ArtisticAda", 
+      email: "ada@rails.com"
+    },
+    {
+      username: "RationalRhys", 
+      email: "rhys@rails.com"
+    }
+  ]
+end
+
+def users
+  users_details.map do |user_details|
+    user = User.new(user_details)
+    user.password = "password"
+    user.password_confirmation = "password"
+    user.skip_confirmation!
+    user.save!
+    user
+  end
+end
+
+def lessons
+  [
+    {
+      project: Lesson.where("is_project = ?", true).third,
+      title: "google_homepage"
+    },
+    {
+      project: Lesson.where("is_project = ?", true).fourth,
+      title: "etch-a-sketch"
+    }
+  ]
+end
+
+users.each do |user|
+  lessons.each do |lesson|
```

Comment:
I think we should add these all to the one project, just to keep things in one place and to ensure that theres enough fake projects in one Project to be able to test what an all submissions page looks like with a lot of projects in it.